tests/pubsub/integration/testgossipsubmeshmanagement.nim

```diff
@@ -334,14 +334,7 @@ suite "GossipSub Integration - Mesh Management":
     # When DValues of Node0 are updated back to the initial dValues
     node0.parameters.applyDValues(dValues)
 
-    # Waiting more than one heartbeat (60ms) and less than pruneBackoff (1s)
-    await sleepAsync(pruneBackoff.div(2))
-    check:
-      node0.mesh.getOrDefault(topic).len == newDValues.get.dHigh.get
```

Comment:
removed because there is no need to check that current values (dValues) are _not_ still applied.